Makefile

```diff
@@ -63,7 +63,7 @@ benchmark-python-local:
 	FEAST_USAGE=False IS_TEST=True FEAST_IS_LOCAL_TEST=True python -m pytest --integration --benchmark  --benchmark-autosave --benchmark-save-data sdk/python/tests
 
 test-python:
-	FEAST_USAGE=False IS_TEST=True python -m pytest -n 8 sdk/python/tests
+	FEAST_USAGE=False IS_TEST=True python -m pytest -n 8 sdk/python/tests -k "not test_sql_registry"
```

Comment:
why did you exclude this test? Can you add a comment?

Reply:
added comment

Reply:
actually, just marked the `test_sql_registry` tests as integration tests since they require external services

Reply:
actually after talking with Achal, we decided to make it a unit test; will do in a separate PR